globus_action_provider_tools/testing/patches.py

```diff
@@ -6,3 +6,8 @@
     "globus_action_provider_tools.flask.api_helpers.TokenChecker.check_token",
     return_value=mock_authstate(),
 )
+
+flask_blueprint_tokenchecker_patch = patch(
```

Comment:
I need to make sure I'm up to speed on proper use of these mocks etc. I want to be sure I'm using them properly and as much as possible in any AP test cases I've created.

Reply:
When I get a chance to revisit I'll add a page in the documentation for how to use the test helpers. There's an example of using them in each of the Flask helper sample apps but formalizing it with a description would be better for sure.